src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -665,6 +724,80 @@ setupSimpleLoopFixpointFeature sym sc sawst cfg mvar func =
 
        return (result_substitution, result_condition)
 
+setupSimpleLoopFixpointCHCFeature ::
+  ( sym ~ W4.B.ExprBuilder n st fs
+  , C.IsSymInterface sym
+  , ?memOpts::C.LLVM.MemOptions
+  , C.LLVM.HasLLVMAnn sym
+  , C.IsSyntaxExtension ext
+  ) =>
+  sym ->
+  SharedContext ->
+  SAWCoreState n ->
+  C.CFG ext blocks init ret ->
+  C.GlobalVar C.LLVM.Mem ->
+  TypedTerm ->
+  IO (C.ExecutionFeature p sym ext rtp, IORef (Crucible.LLVM.FixpointCHC.ExecutionFeatureContext sym 64 ext))
+
+setupSimpleLoopFixpointCHCFeature sym sc sawst cfg mvar func = do
+  let ?ptrWidth = knownNat @64
```

Comment:
I'm not very comfortable with hard-coding this here (which could deviate from similar settings in src/SAWScript/X86Spec.hs), but there's nothing here that looks immediately available for inheriting/determining this value.